composer.json

```diff
@@ -126,7 +126,7 @@
         "doctrine/dbal": "2.9.* <2.9.3",
         "doctrine/doctrine-migrations-bundle": "<1.1",
         "doctrine/orm": "<2.4",
-        "knplabs/knp-time-bundle": "1.9.0",
+        "knplabs/knp-time-bundle": "1.9.0 || >=1.11.0",
```

Comment:
We cannot add a conflict `>=1.11.0`, because that implies that we will never get updates again. Which versions exactly are buggy and in which version has the bug been fixed?

Reply:
Version 1.11 uses Symfony\Contracts, and therefore is not compatible with Symfony 4.2, but Contao 4.8 is. The conflict can be removed in Contao 4.9

Reply:
No, this is wrong (see https://github.com/contao/contao/pull/1140). Please revert the changes and rebase to the latest 4.8 branch.

Reply:
No, this is wrong (see https://github.com/contao/contao/pull/1140). Please revert the changes and rebase to the latest 4.8 branch.

Reply:
Does that work with Symfony 4.2? Implementing an interface that does not exist?

Reply:
The contracts package has been [added in Symfony 4.2](https://symfony.com/doc/4.2/components/contracts.html), so it should work. @ausi /cc